migrations/2019-02-18T214320Z_init/up.sql

```diff
@@ -75,7 +75,7 @@ $$;
 
 -- Add the trigger to recursively delete subresources before a resource delete.
 CREATE TRIGGER resource_path_delete_children
-    BEFORE DELETE ON resource
+    AFTER DELETE ON resource
```

Comment:
(question) Just to make sure - `migrations` will be append-only after it's live on production or merged to master, yes?

Reply:
yes! I'm only messing with this here since we're still working on the db branch, and this was a bug with the original db schema. you're correct, any migrations merged to master shouldn't be changed and the `migrations` directory should be "append-only".

Reply:
I will add a note about this in the migrations readme.

Reply:
Much thanks!